amadeus-parquet/src/internal/record/impls.rs

```diff
@@ -1,4 +1,4 @@
-use hashlink::LinkedHashMap;
+use hashlink::linked_hash_map::LinkedHashMap;
```

Comment:
What's the reason for this?

Reply:
Was getting some strange issues due to double struct use in hashlink lib.rs. Found this fix was necessary to stop the given issue when compiling.